assets/src/edit-story/components/panels/document/publish/publish.js

```diff
@@ -204,63 +156,51 @@ function PublishPanel() {
-            <Media
-              ref={publisherLogoRef}
-              value={publisherLogoUrl}
-              onChange={handleChangePublisherLogo}
-              title={__('Select as publisher logo', 'web-stories')}
-              buttonInsertText={__('Select as publisher logo', 'web-stories')}
-              type={allowedImageMimeTypes}
-              ariaLabel={__('Publisher logo', 'web-stories')}
-              variant={MEDIA_VARIANTS.CIRCLE}
-            />
-          </MediaWrapper>
-        </Row>
         <HighlightRow isHighlighted={highlightPoster?.showEffect}>
-          <LabelWrapper>
-            <FieldLabel>{__('Poster image', 'web-stories')}</FieldLabel>
-            <Required />
-          </LabelWrapper>
-          <MediaWrapper isHighlighted={highlightPoster?.showEffect}>
-            <Media
-              ref={posterButtonRef}
-              value={featuredMedia?.url}
-              onChange={handleChangePoster}
-              title={__('Select as poster image', 'web-stories')}
-              buttonInsertText={__('Select as poster image', 'web-stories')}
-              type={allowedImageMimeTypes}
-              ariaLabel={__('Poster image', 'web-stories')}
-            />
-          </MediaWrapper>
+          <MediaInputWrapper>
+            <MediaWrapper isHighlighted={highlightPoster?.showEffect}>
+              <Sizer width={54} height={96}>
+                <Media
+                  ref={posterButtonRef}
+                  value={featuredMedia?.url}
+                  onChange={handleChangePoster}
+                  title={__('Select as poster image', 'web-stories')}
+                  buttonInsertText={__('Select as poster image', 'web-stories')}
+                  type={allowedImageMimeTypes}
+                  ariaLabel={__('Poster image', 'web-stories')}
+                />
+              </Sizer>
+            </MediaWrapper>
+            <LabelWrapper>
+              <Label>{__('Poster image', 'web-stories')}</Label>
+              <Required />
+            </LabelWrapper>
+          </MediaInputWrapper>
+          <MediaInputWrapper>
+            <MediaWrapper isHighlighted={highlightLogo?.showEffect}>
+              <Sizer width={72} height={72}>
+                <Media
+                  ref={publisherLogoRef}
+                  value={publisherLogoUrl}
+                  onChange={handleChangePublisherLogo}
+                  title={__('Select as publisher logo', 'web-stories')}
+                  buttonInsertText={__(
+                    'Select as publisher logo',
+                    'web-stories'
+                  )}
+                  type={allowedImageMimeTypes}
+                  ariaLabel={__('Publisher logo', 'web-stories')}
+                  variant={MEDIA_VARIANTS.CIRCLE}
+                />
+              </Sizer>
+            </MediaWrapper>
+            <LabelWrapper>
+              <Label>{__('Publisher logo', 'web-stories')}</Label>
+              <Required />
+            </LabelWrapper>
```

Comment:
Font-sizes for these appear wrong. Both are 16px but [should be](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3398%3A94450) 14px and 12px respectively:

| Editor | Figma |
|-|-|
| ![Screen Shot 2021-03-15 at 21 18 21](https://user-images.githubusercontent.com/637548/111241702-0649e880-85d4-11eb-9110-00e943edc2c6.png) | ![Screen Shot 2021-03-15 at 21 19 11](https://user-images.githubusercontent.com/637548/111241742-1f529980-85d4-11eb-92fb-2ecbecf64108.png) |

Also, what's the plan for the _"manage"_ link? Future ticket created?

Reply:
> Font-sizes for these appear wrong.

Ok, will check with UX since supposedly if we need to change the font sizes of a preset, something might be off. 

On the Manage link: We had an issue somewhere for the Setting Page where the editor could choose between the Publisher logos instead of having just one set here. I think [this one](https://github.com/google/web-stories-wp/issues/5108).